File: mario-less/mario.c
#include <stdio.h>
#include <cs50.h>

void pyramid(int height);

int main(void)
{
    //User Input //
    bool input_height = false;
    while(input_height == false)
    {
        int height = get_int("Enter Height: ");
        if(height >= 1 && height <= 8)
        {
            pyramid(height);
            input_height = true;
        }
        else
        {
            input_height = false;
        }
    }
    return 0;
}
void pyramid(int height)
{
    // Looping for Height //
    for(int i = 0 ; i < height ; i++)
    {
        // Looping For Columns //
        for(int j = 1 ; j < (height-i) ; j++)
        {
            printf(" ");
        }
        for(int k = 0 ; k <= i ; k++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
from cs50 import get_int

# Mario- More.py

# Checking Condition
while True:
    # Getting User Input
    height = get_int("Height: ")
    if height in range(1, 9):
        break
    else:
        continue

# Looping For Pyramid
for i in range(1, height + 1):
    # Printing Trailing Spaces
    print(" " * (height - i), end="")
    # Printing First Pyramid Hashes
    print("#" * i, end="")
    # Printing Gap between two pyramids
    print("  ", end="")
    # Printing Second Pyramid Hashes
    print("#" * i)